src/pymatgen/io/cif.py

```diff
@@ -969,8 +969,22 @@ def _get_structure(
         primitive: bool,
         symmetrized: bool,
         check_occu: bool = False,
+        min_vol: float = 0.5,
```

Comment:
The reason I decided to add this arg to `_get_structure` instead of `get_lattice` is that I wish the latter should just read lattice as is without any processing, and let structure parse to decide whether it's valid.

I'm not sure if we want to expose it to `parse_structures` as a public arg though, as this is more a internal sanity check to avoid infinite loop.

Comment welcome as always.